layouts/docs/baseof.html

```diff
@@ -17,10 +17,10 @@
     <div class="container-fluid td-outer">
       <div class="td-main">
         <div class="row flex-md-nowrap">
-          <div class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
+          <div class="col- col- col- td-sidebar d-print-none">
```

Comment:
IMO the removal of the Bootstrap CSS class is causing the site grid layout to break in the deploy preview.

Refer: [Bootstrap Grid system doc](https://getbootstrap.com/docs/4.0/layout/grid/)